neutron_plugin/security_group.py

```diff
@@ -139,3 +150,81 @@ def _rules_for_sg_id(neutron_client, id):
     rules = neutron_client.list_security_group_rules()['security_group_rules']
     rules = [rule for rule in rules if rule['security_group_id'] == id]
     return rules
+
+
+def process_rules(client, sgr_default_values, cidr_field_name,
+                  remote_group_field_name, min_port_field_name,
+                  max_port_field_name):
+    rules_to_apply = ctx.node.properties['rules']
```

Comment:
@kbijakowski What if the ``properties`` does not contain a ``rules`` key what happen on this case ?

Reply:
DONE